tests/e2e/defaultvmgroup_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test suite tries to add, remove and replace vm to the _DEFAULT vmgroup.
+// Expected behavior is that add/rm/replace vm for _DEFAULT vmgroup should fail
+
+package e2e
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	con "github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	ErrorMsg = "This feature is not supported for vmgroup _DEFAULT."
+)
+
+type DefaultVMGroupTestSuite struct {
+	esxIP    string
+	hostName string
+}
+
+func (s *DefaultVMGroupTestSuite) SetUpSuite(c *C) {
+	s.hostName = govc.RetrieveVMNameFromIP(os.Getenv("VM2"))
+	s.esxIP = os.Getenv("ESX")
+	out, err := admincli.ConfigInit(os.Getenv("ESX"))
```

Comment:
Pls. get all values from the inputparams pkg. Thats the single place where env/govc should be used to get values.